<launch>

  <arg name="dir_path" default="$(find bwi_planning)/config/simulation" />
  <arg name="map_file" default="$(find utexas_gdc)/maps/3ne.yaml" />
  <arg name="location_file" default="$(arg dir_path)/locations.yaml" />
  <arg name="door_file" default="$(arg dir_path)/doors.yaml" />

  <arg name="initial_file" default="$(find bwi_planning)/examples/initial" />
  <arg name="query_file" default="$(find bwi_planning)/examples/query" />
  <arg name="domain_semantics_file" default="$(find bwi_planning)/domain/nav" />
  <arg name="rigid_knowledge_file" default="$(find bwi_planning)/domain/example" />

  <!-- launch the planner -->
  <node name="planner_krr2014" type="planner_krr2014" pkg="bwi_planning"
        output="screen">
    <param name="dry_run" value="true" />
    <param name="map_file" value="$(arg map_file)" />
    <param name="door_file" value="$(arg door_file)" />
    <param name="location_file" value="$(arg location_file)" />
    <param name="initial_file" value="$(arg initial_file)" />
    <param name="query_file" value="$(arg query_file)" />
    <param name="domain_semantics_file" value="$(arg domain_semantics_file)" />
    <param name="rigid_knowledge_file" value="$(arg rigid_knowledge_file)" />
    <param name="auto_open_door" value="true" />
  </node>

  <!-- launch the gui -->
  <node name="segbot_gui" pkg="segbot_gui" type="question_dialog_plugin" />

</launch>
